subsys/bluetooth/host/gatt.c

```diff
@@ -961,12 +961,24 @@ struct gatt_chrc {
 	};
 } __packed;
 
+uint16_t bt_gatt_attr_value_handle(const struct bt_gatt_attr *attr)
+{
+	struct bt_gatt_chrc *chrc = attr->user_data;
+	u16_t handle = chrc->value_handle;
+
```

Comment:
We have to make sure the attr given here is really of the same type we expect which you can probably accomplish by checking if the read callback is bt_gatt_attr_read_chrc, also add an assert to check for NULL attribute.

Reply:
done